packages/tracing/src/browser/browsertracing.ts

```diff
@@ -195,8 +195,10 @@ export class BrowserTracing implements Integration {
 
   /** Create routing idle transaction. */
   private _createRouteTransaction(context: TransactionContext): Transaction | undefined {
+    const op = context.op ? `${context.op} ` : '';
```

Comment:
If the `op` isn't set, what's about the transaction's name? would it be undefined as well or isn't that possible? We could concatenate the transaction's name if it's possible.